dev/tasks/generate-crds

```diff
@@ -55,7 +55,7 @@ go run ./scripts/crd-tools reflow-descriptions --dir apis/config/crd/
 
 # excluded_resources are resources that are under development for a direct conversion
 # we don't modify the CRD just yet for those but will in the future
-excluded_resources=("computeforwardingrule")
+#excluded_resources=("computeforwardingrule")
```

Comment:
If the resource is ready, maybe not place it inside `excluded_resources` just in case it's turned back off by accident?

Reply:
Thanks! I made this a part of the comment, so people have an example on how to use this flag when working on direct controllers.

```
# excluded_resources are resources that are under development for a direct conversion
# we don't modify the CRD just yet for those but will in the future
# i.e. excluded_resources=("computeforwardingrule")
```